substrate/frame/staking/src/tests/election_data_provider.rs

```diff
@@ -0,0 +1,433 @@
+use super::*;
```

Comment:
All the new files are missing the Substrate header (see e.g. [here](https://github.com/paritytech/polkadot-sdk/blob/master/substrate/frame/staking/src/slashing.rs#L1-L15) or in any other rs file under `polkadot-sdk/substrate/`).

Reply:
actually had them before I started this all over again 🙈